src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -671,11 +692,15 @@ def __write_status_file(self):
             status_file_payload['status']['substatus'].append(self.__metadata_for_healthstore_substatus_json)
         if self.__configure_patching_substatus_json is not None:
             status_file_payload['status']['substatus'].append(self.__configure_patching_substatus_json)
-        if os.path.isdir(self.status_file_path):
+        if os.path.isdir(self.complete_status_file_path):
             self.composite_logger.log_error("Core state file path returned a directory. Attempting to reset.")
-            shutil.rmtree(self.status_file_path)
+            shutil.rmtree(self.complete_status_file_path)
```

Comment:
missing code coverage?

Reply:
Not sure how to test this, this code is unreachable. do you have any suggestions?